src/adera/Machines/RCSController.h

```diff
@@ -103,21 +103,32 @@ class MachineRCSController : public osp::active::Machine
     std::vector<osp::active::WireOutput*> existing_outputs() override;
 
 private:
-    osp::active::WireInput m_wiCommandOrient;
-    osp::active::WireOutput m_woThrottle;
+    osp::active::WireInput m_wiCommandOrient{this, "Orient"};
+    osp::active::WireOutput m_woThrottle{this, "Throttle"};
+
+    osp::active::ActiveEnt m_rigidBody{entt::null};
 };
```

Comment:
recommend doing // class name of class here at the end of a class to make it easier for reviewers to recognize what class the code is for.